Calendar/BookingHelpers/BookingStaffLogHelper.ps1

```diff
@@ -0,0 +1,38 @@
+﻿function GetStaffMembershipLogs {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$identity
+    )
+    Write-Verbose "Checking Membership logs for $identity"
+    $log = Export-MailboxDiagnosticLogs  -Identity $identity  -ComponentName BookingStaffMembershipLog -ErrorAction SilentlyContinue
+    return $log
+}
+
+
+function GetMembershipLogArray {
+    Write-Verbose "Reading Membership logs for $identity and converting as array"
+    $log = Export-MailboxDiagnosticLogs  -Identity $identity  -ComponentName BookingStaffMembershipLog -ErrorAction SilentlyContinue
+    $alog = createArrayFromStaffLog -log $log.MailboxLog
+    #$alog | Format-Table
+    return $alog
+}
+
+function createArrayFromStaffLog {
+    param([string]$log)
+    $logArray = $log -split [Environment]::NewLine
```

Comment:
>$logArray [](http://example.com/codeflow?start=4&length=9)

Might be nice to have one line from the log as a comment. (i.e. to see/show what you are working with).